lib/actions/logs.ts

```diff
@@ -67,7 +68,12 @@ export const logs: CommandDefinition<
 			$ balena logs 192.168.0.31
 			$ balena logs 192.168.0.31 --service my-service
 			$ balena logs 192.168.0.31 --system
-			$ balena logs 192.168.0.31 --system --service my-service`,
+			$ balena logs 192.168.0.31 --system --service my-service
+
+			$ balena logs 23c73a1.local
+			$ balena logs 23c73a1.local --tail
+			$ balena logs 23c73a1.local --service my-service
+			$ balena logs 23c73a1.local --system --service my-service`,
```

Comment:
Similar to the other comment, maybe 3 examples for each are enough, covering different flags between them.